integrated_channels/blackboard/client.py

```diff
@@ -351,6 +376,46 @@ def generate_post_users_grade_url(self, course_id, column_id, user_id):
             ),
         )
 
+    def generate_create_course_content_url(self, course_id):
+        """
+        Blackboard API url helper method.
+        Path: Course's content objects
+        """
+        return '{base}{path}'.format(
+            base=self.enterprise_configuration.blackboard_base_url,
+            path=COURSE_CONTENT_PATH.format(course_id=course_id)
+        )
+
+    def generate_create_course_content_child_url(self, course_id, content_id):
+        """
+        Blackboard API url helper method.
+        Path: Course content's children objects
+        """
+        return '{base}{path}'.format(
+            base=self.enterprise_configuration.blackboard_base_url,
+            path=COURSE_CONTENT_CHILDREN_PATH.format(course_id=course_id, content_id=content_id)
+        )
+
+    def generate_integration_content_data(self, title, link_to_enrollment, syllabus_body, course_image_url):
+        """
+        Blackboard API json formatting helper method.
+        Format the necessary json to create content data through Blackboard's API.
+        """
+        data = {
+            'title': title,
+            'availability': 'Yes',
+            'contentHandler': {
+                'id': 'resource/x-bb-externallink',
+                'url': link_to_enrollment
+            },
+            'body': '<div>{title}</div><div>{description}</div><img src={image_url} /></p>'.format(
```

Comment:
will this format properly in blackboard side with three divs next to each other? I figure they will handle all layout css etc?

Reply:
"Properly" is hard to define here for me. I've been trying to get Ken to help work up a design. This "displays" all the information, but in a way that I would argue isn't ideal. I'll try and improve upon the design and get Ken's feedback before this is merged.

Reply:
got it, as long as it displays 'well' on Bb side this should be fine.. just wondering if they have any guidance on div classes etc .. but if no, then we are done for this round